package.json

```diff
@@ -35,9 +38,6 @@
     "rimraf": "^2.6.1",
     "ws": "^5.1.1"
   },
-  "puppeteer": {
-    "chromium_revision": "557152"
```

Comment:
Can this be left where it was for diff purposes

Reply:
I moved it up intentionally so that I don't have to scroll editor every time I roll :)